### PR TITLE
Don't format dollar plus curly brackets

### PR DESCRIPTION
**Pull request by BeishaoCao-db**

### CODE REVIEW

File: src/core/Tokenizer.js
Comment (romainr) [inline src/core/Tokenizer.js:106]: Is it the main change? (difficult to see)

**Review by romainr — COMMENTED** [review]
Great fix!

Just wondering before we merge:
- should we avoid all the code styling changes so that it is easier to rebase one day?
- do we want to also do plan a rebase in Q1?

@gabrieldutra

**Comment by gabrieldutra** [comment]
Man, why do we have Restyled acting in this code base 😂. I'd ignore it TBH

**Review by gabrieldutra — APPROVED** [review]
Change LGTM, let's just drop the 2 commits from restyled (it shouldn't be acting in this codebase, only in getredash/redash). Tried to do myself, but you didn't check the option in github to allow push from maintainers ^^

File: src/core/Tokenizer.js
Comment (BeishaoCao-db) [reply src/core/Tokenizer.js:79]: Yes, I revert the styling stuff. Sorry about that

**Review by romainr — COMMENTED** [review]
:1st_place_medal: 

`$$` and others probably similar too :)

**Pull request merged**
